wrappers/here8.rb

```diff
@@ -279,7 +283,10 @@ def build_route_params(dimension, departure_time, arrival_time, lang, with_geome
       {
         routingMode: here_routing_mode(dimension.to_s.split('_').collect(&:to_sym)),
         transportMode: @mode,
-        'avoid[features]': here_avoid_features(options).join(','),
+        avoid: {
```

Comment:
Won't it cause some issue to have some parameters as plain url params and some as body ?

Reply:
I confirm it works like this.